spring-security-rest/build.gradle

```diff
@@ -1,63 +1,35 @@
 ext {
-    pac4jVersion = '4.0.1'
+    pac4jVersion = '4.5.7'
 }
 
 dependencies {
-    compile "org.springframework.boot:spring-boot-starter-actuator"
-    compile "org.springframework.boot:spring-boot-starter-tomcat"
-    compile "org.grails:grails-dependencies"
-    compile "org.grails:grails-web-boot"
+    implementation "org.springframework.boot:spring-boot-starter-actuator"
```

Comment:
Please look at my previous comment. The scope of dependencies for library/plugin should be `api`.